dev-util/unicorn/files/unicorn-2.1.3-strings.patch

```diff
@@ -0,0 +1,40 @@
+https://github.com/unicorn-engine/unicorn/pull/2186
+
+From e57d319971005235cc6946f0ba9e832a339aecb1 Mon Sep 17 00:00:00 2001
+From: "Z. Liu" <zhixu.liu@gmail.com>
+Date: Thu, 15 May 2025 21:50:55 +0800
+Subject: [PATCH] qemu/configure: make strings command can be redefined by
+ ${STRINGS}
```

Comment:
Just nitpicking: Please refer Upstream Pull Request and Gentoo Bugtracker Bugs like follows:

```suggestion
From e57d319971005235cc6946f0ba9e832a339aecb1 Mon Sep 17 00:00:00 2001
From: "Z. Liu" <zhixu.liu@gmail.com>
Date: Thu, 15 May 2025 21:50:55 +0800
Subject: [PATCH] qemu/configure: make strings command can be redefined by
 ${STRINGS}
Upstream: https://github.com/unicorn-engine/unicorn/pull/2186
Bug: https://bugs.gentoo.org/940715
```